__tests__/app.test.js

```diff
@@ -51,3 +51,44 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET 200: responds with an object with a key of article and a list of properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toMatchSnapshot({
```

Comment:
toMatchSnapshot is usually used more in front end testing when we have things that might be changing or reacting to inputs so we can check them later. With things like this, its more reasonable to use things like toMatchObject instead